Svc/FramingProtocol/FprimeProtocol.cpp

```diff
@@ -127,6 +129,7 @@ DeframingProtocol::DeframingStatus FprimeDeframing::deframe(Types::CircularBuffe
     FW_ASSERT(buffer.getSize() >= size);
     buffer.setSize(size);
     ring.peek(buffer.getData(), size, FpFrameHeader::SIZE);
```

Comment:
Do we need to assign "const Fw::SerializeStatus status = ring.peek(...)" in line 131 before asserting in line 132?

Reply:
Yes, good catch. Fixed.